src/components/SiteNavigation.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
This file has been simplified significantly and the only notable diff is the new directory.